recipes/PeoplesSpeech/ASR/transformer/extra_requirements.txt

```diff
@@ -0,0 +1 @@
+datasets
```

Comment:
what about fixing this dep to a specific datasets version?

Reply:
blblbl but which one.

Reply:
No, I don't want to in the end.

Reply:
then face the consequences of having this recipe not working in the next release of HF datasets because they decided to modify the way caching works. enjoy. : p